IFC4x3/Sections/Domain specific data schemas/Schemas/IfcTunnelDomain/Documentation.md

```diff
@@ -0,0 +1,10 @@
+The _IfcTunnelDomain_ schema defines specific object concepts used in the tunnel domain that have not been covered in lower level schemas or schemas in the same level.
```

Comment:
What do you mean by lower level schemas? This is a bit unclear because the official levels below are (in order of appearance), interop, core and resource. I have the feeling you are referring to same level schemas (architecture, road etc.)

Reply:
Hmmm, I copied this text from IFC Rail minus explicit references to other schemas since I don't know the exact list. For Tunnels you can of course use e.g. IfcElectricalDomain, IfcBuldingControlsDomain etc which are also domain specific. Do you have a proposal?

Reply:
My proposal would be to just remove "lower" from "lower level". 😄

Reply:
Better now?

Reply:
Missed one spot. New commit incoming...